src/components/H5PElement/h5pApi.js

```diff
@@ -34,3 +34,15 @@ export const editH5PiframeUrl = async url => {
   );
   return resolveJsonOrRejectWithError(response);
 };
+
+export const fetchH5PTitle = async resourceId => {
+  const url = `${config.h5pApiUrl}/v1/resource/${resourceId}/copyright`;
+  return await fetch(url)
+    .then(resolveJsonOrRejectWithError)
+    .then(values => {
+      return values.h5p.title;
+    })
+    .catch(() => {
+      return null;
+    });
+};
```

Comment:
Du trenger ikke ha await her siden en async funksjon uansett skal returnere et promise. Det fungerer likevel siden returverdien blir wrappet i et promise hvis du ikke returner et selv, men bør ryddes opp i. Jeg ville ha prøvd å skrive om funksjonen til å ligne de to andre i fila.

Reply:
Er det await jeg ikke trenger, eller then?

Reply:
Du trenger ikke begge. Tror det beste er å bruke await på samme måte som i de to andre funksjonene